api/src/main/java/com/exadel/frs/core/trainservice/config/repository/Listener.java

```diff
@@ -0,0 +1,57 @@
+package com.exadel.frs.core.trainservice.config.repository;
+
+import static com.exadel.frs.core.trainservice.system.global.Constants.SERVER_UUID;
+import com.exadel.frs.core.trainservice.dto.DbActionDto;
+import com.exadel.frs.core.trainservice.service.DbActionService;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import lombok.extern.slf4j.Slf4j;
+import org.postgresql.PGConnection;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.scheduling.annotation.Scheduled;
+
+@Slf4j
+class Listener {
+
+    @Autowired
+    private DbActionService actionService;
+
+    private final ObjectMapper mapper = new ObjectMapper();
+
+    private final Connection conn;
+    private final PGConnection pgconn;
+
+    Listener(Connection conn) throws SQLException {
+        this.conn = conn;
+        this.pgconn = (PGConnection)conn;
+        Statement stmt = conn.createStatement();
+        stmt.execute("LISTEN updatemsg");
```

Comment:
Is updatemsg a name of the notify/listen channel? 
If yes, please rename it to something understandable like "face_collection_update_msg"